src/libs/actions/PaymentMethods.ts

```diff
@@ -269,21 +285,20 @@ function dismissSuccessfulTransferBalancePage() {
 
 /**
  * Looks through each payment method to see if there is an existing error
- * @param {Object} bankList
- * @param {Object} fundList
- * @returns {Boolean}
+ *
  */
-function hasPaymentMethodError(bankList, fundList) {
+function hasPaymentMethodError(bankList: OnyxValues[typeof ONYXKEYS.BANK_ACCOUNT_LIST], fundList: OnyxValues[typeof ONYXKEYS.FUND_LIST]): boolean {
     const combinedPaymentMethods = {...bankList, ...fundList};
-    return _.some(combinedPaymentMethods, (item) => !_.isEmpty(item.errors));
+
+    return Object.values(combinedPaymentMethods).some((item) => !!item.errors);
 }
 
 /**
  * Clears the error for the specified payment item
- * @param {String} paymentListKey The onyx key for the provided payment method
- * @param {String} paymentMethodID
+ * @param paymentListKey The onyx key for the provided payment method
+ * @param paymentMethodID
  */
-function clearDeletePaymentMethodError(paymentListKey, paymentMethodID) {
+function clearDeletePaymentMethodError(paymentListKey: typeof ONYXKEYS.BANK_ACCOUNT_LIST | typeof ONYXKEYS.FUND_LIST, paymentMethodID: string) {
```

Comment:
extract `typeof ONYXKEYS.BANK_ACCOUNT_LIST | typeof ONYXKEYS.FUND_LIST` to a type